delocate/tools.py

```diff
@@ -135,6 +135,10 @@ def _line0_says_object(line0, filename):
     if line0.startswith('Archive :'):
         # nothing to do for static libs
         return False
+    if line0.endswith("'%s': The file was not recognized as a valid object file." % filename):
```

Comment:
I added another test for the second message (printed when used with an `ico` file. The first message is already tested using [a non binary file](https://github.com/matthew-brett/delocate/pull/14/files#diff-0ad957b46ea71811bdc0e07301e69736R42).

```
$ otool -L delocate/tests/data/icon.ico 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/objdump: 'delocate/tests/data/icon.ico': The end of the file was unexpectedly encountered.
fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: internal objdump command failed
```

```
$ otool -L delocate/tests/test_install_names.py
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/objdump: 'delocate/tests/test_install_names.py': The file was not recognized as a valid object file.
fatal error: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/otool: internal objdump command failed
```